.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This config tells ESlint how to lint your code base. You can add one to any JS/TS project by running `npx eslint --init` and filling in the questionnaire. [eslint](https://eslint.org/)